programs/fileio.c

```diff
@@ -800,10 +830,10 @@ static void FIO_adjustMemLimitForPatchFromMode(FIO_prefs_t* const prefs,
  *         If neither flag is specified, zstd will prompt the user for confirmation to proceed.
  * If --rm is not specified, then zstd will print a warning to the user (which can be silenced with -q).
  */
-static int FIO_removeMultiFilesWarning(const FIO_prefs_t* const prefs, int displayLevelCutoff, const char* outFileName)
+static int FIO_removeMultiFilesWarning(const FIO_prefs_t* const prefs, FIO_ctx_t* const fCtx, int displayLevelCutoff, const char* outFileName)
```

Comment:
minor :
argument order (programming pattern)

Prefer having the object _state_ (when it exists) as the first argument in the list, in this case `fCtx`.
Then followed by mutable arguments when they exist (such as `void* dst`), then immutable ones.
`prefs` belong to the read-only parameter category, and should get further down the list of parameters.

It's merely a pattern we use everywhere.
Obviously, it doesn't change a thing to the way the function works.